gulpfile.js

```diff
@@ -915,6 +927,85 @@ function build() {
   return performBuild().then(() => exitCtrlcHandler(handlerProcess));
 }
 
+/**
+ * Starts a nailgun server (provides a fast-running closure compiler instance),
+ * and replaces google-closure-compiler's binary with the nailgun runner
+ */
+async function startNailgunServer() {
+  if (argv.single_pass) {
+    return;
+  }
+  // Replace default binary with nailgun on linux and macos
+  if (process.platform == 'darwin') {
+    nailgunRunnerReplacer = require('require-hijack')
```

Comment:
Did you see https://www.npmjs.com/package/google-closure-compiler#running-the-compiler-using-nailgun?

Reply:
Yes, I saw it, tried it, and couldn't get it to work with our custom runner, most likely because the version of [`closure-gun`](https://www.npmjs.com/package/closure-gun) linked from the article hasn't seen an update in 2 years. The underlying tool they mention is [`nailgun`](https://github.com/facebook/nailgun), which is exactly what I'm using in this PR, so I believe we're good here.